configs/config/pretrain/vision_transformer/swav/swav_deit_b_2nodes.yaml

```diff
@@ -13,19 +12,18 @@ config:
     NUM_DATALOADER_WORKERS: 8
     TRAIN:
       DATA_SOURCES: [disk_folder]
-      #      DATA_PATHS: ["<path to train folder>"]
-      LABEL_SOURCES: [disk_folder]
-      DATASET_NAMES: [imagenet1k_debug_folder]
+      DATASET_NAMES: [imagenet1k_folder]
       LABEL_TYPE: "zero"
       BATCHSIZE_PER_REPLICA: 16
+      DROP_LAST: True
       TRANSFORMS:
         - name: ImgPilToMultiCrop
           total_num_crops: 2
           size_crops: [224]
           num_crops: [2]
           crop_scales: [[0.14, 1]]
         - name: RandomHorizontalFlip
-        - name: VisslRandAugment
+        - name: RandAugment
```

Comment:
we should rebase to master. This change should be essentially reverted after the rebase.